src/components/achievements/ProgressCollapsableHeadline.tsx

```diff
@@ -0,0 +1,70 @@
+import { ChevronDownIcon, HStack, Text, useBreakpointValue, Box, Link } from 'native-base';
+import { useTranslation } from 'react-i18next';
+import { AchievementState, AchievementType } from '../../types/achievement';
+import { useMemo } from 'react';
+import { Pressable } from 'react-native';
+
+type ProgressCollapsableHeadlineProps = {
+    achievementState?: AchievementState;
+    achievementType?: AchievementType;
+    onClick: () => void;
+};
+
+const ProgressCollapsableHeadline: React.FC<ProgressCollapsableHeadlineProps> = ({ achievementState, achievementType, onClick }) => {
+    const { t } = useTranslation();
+    const headline = useMemo(() => {
+        if (achievementType === AchievementType.STREAK) {
+            return t('achievement.progress.streak.headline');
+        }
+        switch (achievementState) {
+            case AchievementState.COMPLETED:
+                return t('achievement.progress.state.completed');
+            case AchievementState.ACTIVE:
+                return t('achievement.progress.state.active');
+            case AchievementState.INACTIVE:
+                return t('achievement.progress.state.inactive');
+            default:
+                return '';
+        }
+    }, [achievementState, achievementType]);
+    const isCollapsable = useBreakpointValue({
+        base: true,
+        md: false,
+    });
+    const handlePress = () => {
+        isCollapsable && onClick();
+    };
+    return (
+        <Pressable
+            onPress={handlePress}
+            style={{
+                width: '100%',
+                flexDirection: 'column',
+            }}
+        >
```

Comment:
I guess we have to `disable` the pressable on desktop. 
```suggestion
            disabled={!isCollapsable}
```